pkg/kdump/kdump-view.jsx

```diff
@@ -263,6 +318,9 @@ export class KdumpPage extends React.Component {
         super(props);
         this.handleTestSettingsClick = this.handleTestSettingsClick.bind(this);
         this.handleSettingsClick = this.handleSettingsClick.bind(this);
+        this.handleAutomationClick = this.handleAutomationClick.bind(this);
+        this.os_release = null;
+        read_os_release().then(os_release => { this.os_release = os_release });
```

Comment:
this should be a setState() call (well, via useState()), to re-render the page once os-release got read. Otherwise this is a race condition.